### DISCUSSION TITLE
Measuring the real FPS

### DISCUSSION THREAD

**Discussion by Ashir555** [86Box/86Box#2698] [Q&A]
Hi, I'm sorry for bothering you guys again but I used PEcm as a way to "feel" what it was like to play on my first Pentium PC (166 MHz, 32 MB RAM, 2 GB HDD, 1 MB VRAM).

Now, I have Riva Tuner Statistics and Fraps to monitor the FPS on games. On PCEm I could play NFS III and I see that it was slow, around 20-25 FPS, and indeed, the game feel that slow.

But, on 86Box both programs always shows 60 FPS, even when I know and I feel that is not even near that speed.

I try every renderer and all show the same FPS. It's like they are not really measuring the video but what 86Box is telling them the refresh rate is.

Why is this?, is there a way for measure the real FPS a game or program is having running inside 86Box?

**Comment by OBattler** [comment]
If you're using Windows as your host OS, try compiling a build of 86Box with the Win32 UI and see if the same problem occurs there as well.

**Reply by Ashir555** [reply]
I didn't compile but download a Windows 32 bits binary available from github and same result, keep showing 60 FPS...

**Reply by OBattler** [reply]
By Win32, I meant the Win32 UI, not simply a 32-bit Windows build. You have to compile that yourself (give cmake USE_QT = OFF, I belive).
